lib/engine/game/g_18_norway/steps/dividend.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/dividend'
+require_relative '../../../step/half_pay'
+require_relative '../../../step/minor_half_pay'
+
+module Engine
+  module Game
+    module G18Norway
+      module Step
+        class Dividend < Engine::Step::Dividend
+          DIVIDEND_TYPES = %i[payout half withhold].freeze
+          include Engine::Step::HalfPay
+
+          def share_price_change(entity, revenue = 0)
+            return {} if entity.minor?
```

Comment:
There's another reference to minors here, do these exist?

Reply:
No there is not any minors in 18Norway, did copy a bit to much from 1846
Now removed

Also removed receivership